standard/requirements/monitoring-event-topic/REQ_publishing.adoc

```diff
@@ -3,10 +3,9 @@
 |===
 ^|*Requirement {counter:req-id}* |*/req/monitoring-event-topic/publishing*
 ^|A |Events SHALL NOT be published with a topic that is not defined in this specification.
-^|B |Events SHALL be published to exactly level 5.
+^|B |Events SHALL be published to exactly level 4.
 ^|C |Event topic level 1 SHALL be named ``monitor``.
 ^|D |Event topic level 2 SHALL be named ``a``.
 ^|E |Event topic level 3 SHALL be named ``wis2``.
 ^|F |Event topic level 4 SHALL be a centre identifier (as per the <<wis2-topic-hierarchy, WIS2 Topic Hierarchy>>) based on the entity producing the event.
```

Comment:
^|F |Event topic level 4 SHALL be a centre identifier (as per the <<wis2-topic-hierarchy, WIS2 Topic Hierarchy>>) based on the entity subject of the event.